openmls/src/group/public_group/validation.rs

```diff
@@ -423,6 +423,11 @@ impl PublicGroup {
             if !self.treesync().is_leaf_in_tree(removed) {
                 return Err(ProposalValidationError::UnknownMemberRemoval);
             }
+
+            // removed node can not be blank (check 701)
```

Comment:
It would be nice if we'd had anchors in the dashboard to link back here. Do you think that's easily doable?

Reply:
Yeah that would be nice. Options I see:

- make each check a public function and link to the function's rustdoc
  - (+) relatively easy
  - (-) not quite the same as linking to the code
- decide on an anchor string pattern and use text fragments to link to the specific place
  - (+) relatively easy 
  - (-) [doesn't work in firefox, but might come soon](https://bugzilla.mozilla.org/show_bug.cgi?id=1753933)
  - (-) doesn't tolerate if the check moves to a different file
- decide on an anchor string and link to the github search for that string
  - (+) relatively easy 
  - (+) works everywhere
  - (-) might not work for everyone: I think you can only search when you are logged into github
    - not sure that is a problem, this is primarily an internal tool anway

Maybe we should link to github search for now and possibly pivot to text fragment links once firefox supports them.

Reply:
https://github.com/openmls/validation.dhall/pull/13